implementation.md

```diff
@@ -2,4 +2,66 @@
 layout: default
```

Comment:
Is this supposed to be here? It looks strange in the rendered view...

Reply:
Otherwise, LGTM

Reply:
Yeah, this is [Jekyll front matter](https://jekyllrb.com/docs/front-matter/).